WordPress/src/androidTest/java/org/wordpress/android/e2e/DashboardTests.kt

```diff
@@ -1,23 +1,28 @@
 package org.wordpress.android.e2e
 
 import dagger.hilt.android.testing.HiltAndroidTest
+import org.junit.Assume.assumeTrue
 import org.junit.Before
 import org.junit.Test
 import org.wordpress.android.e2e.pages.MySitesPage
 import org.wordpress.android.support.BaseTest
+import org.wordpress.android.test.BuildConfig
 
 @HiltAndroidTest
 class DashboardTests : BaseTest() {
     @Before
     fun setUp() {
+        // We run the class for JP only (so far the class contains
+        // only a test for Domains card, which in not valid for WP)
+        assumeTrue(BuildConfig.IS_JETPACK_APP)
+
```

Comment:
All changes in this file is a cherry-pick from #18349

Reply:
did you mean to link to a different PR/commit? currently that link is this PR

Reply:
Yes, sorry, I mean #18365

Reply:
> I meant https://github.com/wordpress-mobile/WordPress-Android/pull/18365

Are you sure? That PR wasn't yet merged.

Reply:
I'm sorry, @ovitrif. I hope third time's a charm 😅: #18345